junit5-extension/src/test/java/io/kroxylicious/testing/kafka/junit5ext/StaticFieldExtensionTest.java

```diff
@@ -48,47 +47,47 @@ public void testKafkaClusterStaticField()
     }
 
     @Test
-    public void adminStaticField() throws ExecutionException, InterruptedException {
+    void adminStaticField() throws ExecutionException, InterruptedException {
```

Comment:
I don't like to see tests enumerating their checked exceptions. It is just clutter.  In general I prefer to see `throws Exception` when testing an API that throws a checked exception.  However, in this case, I'd have  `assertSameCluster` throw a RuntimeException.

Reply:
Yeah, I think `assertSameCluster` should catch and re-throw as runtime exceptions. However I don't want to pull that thread in this PR. I'm far enough down the rabbit hole from where I started without doing that right now.